examples/website/src/shared/route/articleTemplateContainer.tsx

```diff
@@ -422,7 +422,11 @@ export function PeerProfileBlock({peer, article}: PeerProfileBlockProps) {
         <div className={css(PeerProfileFiller)} />
       </div>
       <div className={css(PeerProfileCallToActionURL)}>
-        {peer?.callToActionText?.length && (
+        {peer?.callToActionString ? (
+          <a target="_blank" rel="noreferrer" href={peer?.callToActionURL}>
+            {peer?.callToActionString}
+          </a>
+        ) : (
           <a target="_blank" rel="noreferrer" href={peer?.callToActionURL}>
```

Comment:
Wouldn't the `callToActionText` also be possible to be empty here? Shouldn't the `callToActionText` and `callToActionString` both be rendered conditionally, independently?